Apps/Sandcastle/gallery/High Dynamic Range.html

```diff
@@ -34,6 +34,10 @@
     shadows : true
 });
 
+if (!viewer.scene.highDynamicRangeSupported) {
+    throw new Cesium.RuntimeError('This browser does not support high dynamic range.');
```

Comment:
Instead of throwing use `console.log`.

Could you edit `Ground Polyline Material`, `Ground Polyline Materials` and `Polylines On Terrain` in the development folder to use `console.log` too?